topics/mvvmcross/index.md

```diff
@@ -0,0 +1,23 @@
+---
+aliases: monocross
+created_by: Alexandru Stefan Popescu
+display_name: MvvmCross
+github_url: https://github.com/mvvmcross
+logo: mvvmcross.png
+released: June 2013
+short_description: The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows and Mac.
+topic: mvvmcross
+url: https://www.mvvmcross.com/
+---
+MvvmCross is a cross-platform MVVM framework that enables developers to create powerful cross platform apps. It supports Xamarin.iOS, Xamarin.Android, Xamarin.Mac, Xamarin.Forms, Universal Windows Platform (UWP), and Windows Presentation Framework (WPF).
```

Comment:
You could spell out "MVVM framework" as "model-view-view model framework".

You could also expand on what kind of framework this is. For example, I see on the website that it's for building mobile apps.

Reply:
All suggestions were applied. Sorry for missing the comma from short description.